components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -3612,9 +3612,12 @@ subroutine ocn_compute_land_ice_flux_input_fields(layerThickness, normalVelocity
       do iCell = 1, nCellsAll
          ! the friction velocity is the square root of the top drag + variance of tidal velocity
          ! (computed regardless of land-ice coverage)
+
          landIceFrictionVelocity(iCell) = sqrt(landIceTopDragCoeff * &
                                    (2.0_RKIND * kineticEnergyCell(minLevelCell(iCell),iCell) &
-                                   + config_land_ice_flux_rms_tidal_velocity**2))
+                                   + config_land_ice_flux_tidal_Jourdain_alpha**2 * &
+                                   (config_land_ice_flux_tidal_Jourdain_A0 * velocityTidalRMS(iCell) + &
+                                   config_land_ice_flux_tidal_Jourdain_U0)**2))
```

Comment:
This is the only term I could imaging causing the non-BFB results.  It's hard for me to understand how a reasonable compiler doesn't produce BFB results when `config_land_ice_flux_tidal_Jourdain_alpha = 1.0`, `config_land_ice_flux_tidal_Jourdain_A0 = 0.0` and `config_land_ice_flux_tidal_Jourdain_U0` is identical to the old `config_land_ice_flux_rms_tidal_velocity` but I guess gnu on pm-cpu might just not be reasonable...

Reply:
That was my thought as well -- but I did want to point it out just in case